Components/home.css

```diff
@@ -0,0 +1,298 @@
+body {
+    height: 312vw;
+    max-width: 100%;
+    overflow-x: hidden;
+}
+
+.nav-bar {
+  min-height: 12vh;
+  width: 100%;
+  align-items: center;
+  background-image: linear-gradient(to right, #D88090 ,#F5A6BB,#F7ADBE, #F5A6BB, #EF95AE);
+  font-weight: 600;
+  color: var(--white);
+  padding: 1rem;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: -2rem;
+}
+
+.list-inline-item {
+  margin-right: 2rem;
+  font-size: 1.8rem;
+  padding-right: 2rem;
+}
+
+.list-inline-icons {
+  margin-right: 2rem;
+  font-size: 1.8rem;
+  padding-right: 0rem;
+}
+
+.nav-section {
+  display: flex;
+  align-items: center;
+}
+
+.nav-section-items {
+  padding: 0 2rem;
+}
+
+.nav-bar-links {
+  display: flex;
+}
+
+.nav-icons {
+  display: flex;
+  justify-self: right;
+}
+
+.nav-logo {
+  width: 4rem;
+  height: 4rem;
+}
+
+.nav-icon-link {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0 0.5rem;
+}
+
+.nav-icon-link .nav-icon-text {
+  font-size: 0.8rem;
+  margin-top: 0.5rem;
+  color: var(--white)
+}
+
+.nav-icon-link .nav-icon {
+  color: var(--white);
+  width: 2rem;
+  height: 2rem;
+  font-size: 2rem;
+}
+
+.search-bar {
+  width: 100%;
+}
+
+input:focus {
+  outline: none;
+}
+
+.search-bar {
+  background-color: var(--white);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: var(--pink-dominant);
+  border-radius: 5rem;
+  justify-self: right;
+}
+
+.search-bar-input {
+  width: calc(100% - 2rem);
+  border: none;
+  background-color: transparent;
+  padding: 1rem 1rem;
+  border-radius: 5rem;
+}
+
+.search-bar-btn {
+  padding: 1rem 0rem 1rem 1rem;
+}
+
+.nav-icon:hover {
+  color: var(--pink-mild);
+  font-size: 2.2rem;
+}
+
+.nav-icon-text {
+  color: var(--gary-light);
```

Comment:
ig spelling of varible grey is wrong

Reply:
Omg! and I thought why it's not working 😅